shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/RouteCardData.kt

```diff
@@ -120,11 +122,38 @@ data class RouteCardData(
         override val upcomingTrips: List<UpcomingTrip>,
         val alertsHere: List<Alert>,
         val allDataLoaded: Boolean,
-        override val hasSchedulesToday: Boolean,
+        val hasSchedulesTodayByPattern: Map<String, Boolean>,
```

Comment:
This is one of my least favorite changes in this PR, but it seemed like the most straightforward path to determining whether we should show "Predictions Unavailable" for a branch with no service vs. if we are dealing with a no scheduled service / service ended scenario.